metadata-ingestion-modules/airflow-plugin/tests/integration/test_plugin.py

```diff
@@ -491,6 +500,21 @@ def test_airflow_plugin(
             ],
         )
 
+        if test_case.multiple_connections and not is_v1:
```

Comment:
this is a bit strange - from the config, we always configure airflow to use a composite emitter. but we only check it if `multiple_connections` is set?

Reply:
I think this comment still applies